### PR TITLE
#12605 - replace Persons involved with Event Participants

### LINKED ISSUE #12605: Rename persons involved to event participants in mobile app [1]
### Problem Description

Event participants are called "persons involved "in the mobile app.

### Proposed Change

To rename all instances or persons involved to event participants or vice versa.

### Added Value/Benefit

Users will not get confused with namings when they use mobile and web apps at the same time 

### Acceptance Criteria

- [ ] All occurrences of "Person involved" has been renamed to "Event participant"
- [ ] All occurrences of "Persons involved" has been renamed to "Event participants"

### Implementation Details

_No response_

### Mockups

_No response_

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by carina29**
Fixes #12605

### CODE REVIEW

**Comment by carina29** [comment]
**This ticket should be merged after 1.91 Release.**

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=12657

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=12657

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=12657

File: sormas-app/app/src/main/java/de/symeda/sormas/app/backend/event/Event.java [hunk 1]
```diff
@@ -75,7 +75,7 @@ public class Event extends PseudonymizableAdo {
 	public static final String EVENT_INVESTIGATION_STATUS = "eventInvestigationStatus";
 	public static final String EVENT_INVESTIGATION_START_DATE = "eventInvestigationStartDate";
 	public static final String EVENT_INVESTIGATION_END_DATE = "eventInvestigationEndDate";
-	public static final String EVENT_PERSONS = "eventPersons";
+	public static final String EVENT_PARTICIPANTS = "eventParticipants";
```
Comment (MateStrysewske) [inline sormas-app/app/src/main/java/de/symeda/sormas/app/backend/event/Event.java:78]: This is now inconsistent with the name of the field in the same class; you have to rename that to eventParticipants as well

**Review by MateStrysewske — CHANGES_REQUESTED** [review]
Please revert the changes to all the translation files in the mobile app (except the default values.xml file); the changes should automatically be done by Crowdin, and Android should fall back to the English default if it can't find the translation in the respective language file

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=12657

File: sormas-app/app/src/main/java/de/symeda/sormas/app/backend/event/Event.java (hunk 1 above)
Comment (carina29) [reply sormas-app/app/src/main/java/de/symeda/sormas/app/backend/event/Event.java:78]: In this Event.java class ( from sormas-app/app/src/main/java/de/symeda/sormas/app/backend/event/Event.java) there is no eventPerson or eventParticipant field. 

In Event.java from sormas-backend/src/main/java/de/symeda/sormas/backend/event/Event.java we had an eventPersons which I've renamed to eventParticipants or are you reffering to something else? :) 

image

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=12657

**Review by MateStrysewske — APPROVED** [review]

**Pull request merged**